src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -94,6 +98,12 @@ def sql_alter_from(self, target_table_key, ws_id):
             f" , '{self.UPGRADED_FROM_WS_PARAM}' = '{ws_id}');"
         )
 
+    def sql_table_in_mount_alter_from(self, target_table_key):
```

Comment:
add a type hint for an argument of the public method. it's not clear what the key is